go/vt/vtgate/executor.go

```diff
@@ -332,9 +332,14 @@ func (e *Executor) handleSavepoint(ctx context.Context, safeSession *SafeSession
 	for i := range rss {
 		queries[i] = &querypb.BoundQuery{Sql: sql}
 	}
+	stmt, err := sqlparser.Parse(sql)
```

Comment:
This is only called from the new executor code, and should be able to just take as input parameter a `ignoreMaxMemoryRows bool`, instead of having to parse and check.

Reply:
Good catch! Resolving this since I updated this to take `ignoreMaxMemoryRows` as a param